xwiki-platform-tools/xwiki-platform-tool-configuration-resources/src/main/resources/xwiki.properties.vm

```diff
@@ -725,4 +730,4 @@ extension.oldflavors=$oldFlavor.trim()
 # debug.minify=false
 
 
-$!xwikiPropertiesAdditionalProperties
```

Comment:
It's not necessarily a mistake, it's just adding a newline at the end of the file. I'd be +1 for that, it makes it a bit easier to add new settings at the end.